### PR TITLE
gh-pages updater

### PR DESCRIPTION
**Pull request by RomanTsegelskyi**
Automatic update of gh-pages after commit. Additional ideas:
- [ ] update only on change of certain files
- [x] update `inst/doc` if vignettes were updated
- [x] update `README` if `README.brew` was updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,9 +115,9 @@ Which command produces the following output by default:
 -------------------------------------------
       &nbsp;         mpg   cyl   disp   hp 
 ------------------- ----- ----- ------ ----
-   **Mazda RX4**     21     6    160   110 
+   **Mazda RX4**    21.0    6    160   110
```
Comment (daroczig) [inline README.md]: We switched on the `keep.trailing.zeros` setting somewhere?

File: .gh-pages-update.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+rm -rf out || exit 0;
+rm -rf master.zip
+rm -rf pander
+rm -rf pander-master
+
+GH_REPO="@github.com/RomanTsegelskyi/pander.git"
```
Comment (daroczig) [inline .gh-pages-update.sh]: This might need to be updated after merge, right?

File: README.md (hunk 1 above)
Comment (RomanTsegelskyi) [inline README.md]: We didn't that seems to be my mistake. Turns out `format` works differently on a vector and value
```r
> format(c(21.0, 21.0, 22.8), digits = 4)
[1] "21.0" "21.0" "22.8"
> format(21.0, digits = 4)
[1] "21"
```
I fixed it in https://github.com/Rapporter/pander/commit/9f01b90a6327082dc0e636515d4e075a73e9488c

File: .gh-pages-update.sh (hunk 2 above)
Comment (RomanTsegelskyi) [inline .gh-pages-update.sh]: Yes, I will do small updated right before merge. Since I have write access to repo, my token should work fine

**Comment by RomanTsegelskyi** [comment]
This seems to be final version. Don't mind the README, it will get updated properly after merge. Good to merge?

**Comment by daroczig** [comment]
It seems to me that only the `gh-pages` branch is updated, but e.g. not the README.md in the `master` branch -- shown on GH. Maybe we should add some checks if `README.brew` changed, then the `md` file should be updated.

Also, can we add reference to the vignettes? E.g. http://rapporter.github.io/pander/evals.html

And last question: these automatic commits are rater large, as the plots are regenerated and **added** to the repo -- instead of reusing the previous files. This is due to the file name global option, which checks the file names in the current directory and generates a file name with increased counter based on `%N`. Quick fix for this: remove all images from the `plot` folder before regenerating the images, so the exact same plots will be regenerated -- none of those added/updated in the repo.

**Comment by RomanTsegelskyi** [comment]
There is a check to update `README.md` when `README.brew` is update (https://github.com/Rapporter/pander/blob/master/.gh-pages-update.sh#L23). But it seems that there is a small typo in regex.

I will add references to all vignette today, when I submit a pull request to update `README.brew`. As for plots, I didn't want to clear `plots` folder since there are 3 plots used by `evals` vignette, but clearly that was a mistake, I will do a smarter regex on what to delete.

**Pull request merged**